### ISSUE TITLE
How to Dump?

### ISSUE THREAD

**Issue by r00tb3** [DU-Fork/RMX1851_Dump#1]
Hey, I'd like to know how did you dump all these files?

Is it from an OTA update or from your device?

Regards.

**Comment by 4j17h** [comment]
yes it is from ozip files which realme use to roll out updates to device. Since ozip files are encrypted, we use https://github.com/bkerler/oppo_ozip_decrypt to decrypt ozip files. Although for dumping we have an automated script which will take care of decrypting ozip files & dumping too, check out https://git.rip/dumps/dumpyara.

**Comment by r00tb3** [comment]
> yes it is from ozip files which realme use to roll out updates to device. Since ozip files are encrypted, we use https://github.com/bkerler/oppo_ozip_decrypt to decrypt ozip files. Although for dumping we have an automated script which will take care of decrypting ozip files & dumping too, check out https://git.rip/dumps/dumpyara.

Thanks a lot... this is very helpful.
